include/VoidAccessor.h

```diff
@@ -43,6 +43,9 @@ namespace ChimeraTK {
     VoidAccessor(Module* owner, const std::string& name, VariableDirection direction, std::string& unit,
```

Comment:
Shouldn't we deprecate this here, too?

Reply:
Well, it's protected and hence not (easily) accessible by user code. It is only used by deprecated accessors of the VoidOutput and VoidInput classes. Hence I think we can avoid the deprecation (which would also involve disabling the deprecation warning where it is legitimately used).

Reply:
Ah. I missed that it is protected.